### PR TITLE
DG-25646: Add docker to the web application

### PR DESCRIPTION
**Pull request by ZLamine**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.idea
+*.class
+build
+*.iml
```
Comment (aarbi) [inline .dockerignore]: it is just a common practice, files should end with an empty new line

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FROM openjdk:8
+
+WORKDIR /opt/code
+COPY . .
+EXPOSE 80:8080
```
Comment (aarbi) [inline Dockerfile]: For visibility, EXPOSE should be moved up just right after the FROM statement

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+CMD ./gradlew bootRun
```
Comment (aarbi) [inline Dockerfile]: A new line here as well

**Review by aarbi — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (ZLamine) [reply Dockerfile]: ok

File: .dockerignore (hunk 1 above)
Comment (ZLamine) [reply .dockerignore]: ok cool

**Review by MeriemSelmi — APPROVED** [review]

**Review by halloulpsx — APPROVED** [review]
LGTM

**Pull request merged**